index.html

```diff
@@ -298,10 +298,12 @@ <h3>Requirements for advancing the maturity level of features</h3>
     its implementability has been demonstrated.</p>
     <p>When considering suitability to advance any feature beyond
     Candidate Recommendation, at least two independent factors of verification
-    MUST be demonstrated, which may come from any of:</p>
+    for each normative requirement
+    MUST be demonstrated, which may come, as relevant for that requirement,
+    from any of:</p>
     <ul>
       <li>Presentation implementation</li>
-      <li>Content</li>
+      <li>Content producing implementation</li>
```

Comment:
Suggestion from the TTWG call 2022-11-25 that an acceptable alternative would be "Content implementation" - @rhiaro please could you confirm if this it the case?

Reply:
"Content implementation" strikes me as ambiguous—it could be interpreted to include hand-written content, which AIUI the council was trying to prevent with the proposed wording.

Reply:
Code is hand-written as well. I am not sure what distinction you are trying to make.

Reply:
(to be clear, my above comment, and this, are with my Apple AC rep hat on, and no other.)

@palemieux the distinction is that a piece of code that generates content would count but human-written content would not. I'm unsure how to be more clear.

Reply:
@hober I'd discussed with the WG on a call on 11/24 that "content implementation" would mean content produced by some form of authoring tool in their context, so including the word "producing" in here wouldn't be necessary. Are you okay with that?

Reply:
Done in 2ba8ed2.

Reply:
> I'm going to go ahead and assume silence is assent, and remove the word "producing".

No, silence just means I didn't see the ping in my firehose of GitHub notifications.

I still think "content implementation" is too ambiguous.